mmcv/utils/progressbar.py

```diff
@@ -44,6 +44,8 @@ def start(self):
     def update(self):
         self.completed += 1
         elapsed = self.timer.since_start()
+        if elapsed == 0.0:
+            elapsed = 0.000001
```

Comment:
Timer needs to be precise and I suggest the following. 
```python
if elapsed > 0:
    fps = self.completed / elapsed
else:
    fps = float('inf')
```

Reply:
Thanks for your comments. I have pushed the new commit.